config/app.php

```diff
@@ -154,6 +154,7 @@
         /*
          * Package Service Providers...
          */
+        \Torann\GeoIP\GeoIPServiceProvider::class,
```

Comment:
@c0repwn3r this should be `Torann\GeoIP\GeoIPServiceProvider::class,`

Reply:
Fixed